rs/execution_environment/benches/run-benchmark.sh

```diff
@@ -21,9 +21,10 @@ TMP_FILE="${TMP_FILE:-${MIN_FILE%.*}.tmp}"
 bash -c "set -o pipefail; \
     bazel run '${BENCH}' -- ${FILTER} ${BENCH_ARGS} \
         2>&1 | tee '${LOG_FILE}' | rg '^(test .* )?bench:' --line-buffered \
-        | sed -uE \
-            -e 's/^test (.+) ... bench: +([0-9]+)...... ns\/iter [(].*/> bench: \1 \2 ms\/iter/' \
-            -e 's/^test (.+) ... bench: +([0-9]+)... ns\/iter [(].*/> bench: \1 \2 µs\/iter/' \
+        | awk '{
+                match(\$0, /^test (.+) ... bench: +([0-9]+) ns\/iter.*/, r)
+                printf \"> %s %.1f ms\n\", r[1], r[2] / 1000 / 1000; fflush();
```

Comment:
This means that 1 microsecond is now displayed as 0.001 ms: does "better readability of shorter runtimes" from the PR description mean equal units or values with not many decimals?

Reply:
It's just a filter to make the nanosecond results more readable while the tests are running. The results are still stored and compared in nanoseconds.

It used to just "cut" the last few digits to display the result in ms or µs, which was also a bit confusing. Now we reduced the working set from 1 GiB to 102 MiB, meaning there are more benchmarks which take just a couple of ms, so displaying all the results in milliseconds and showing some digits after the decimal point seems like an improvement.

Reply:
I've changed it to display two digits.